storybook/manager-head.html

```diff
@@ -0,0 +1,35 @@
+<script>
+	( function redirectIfStoryMoved() {
+		const REDIRECTS = [
+			{
+				from: /\/components-deprecated-/,
+				to: '/components-',
+			},
+		];
+
+		const params = new URLSearchParams( window.location.search );
+
+		const matchedRedirect = REDIRECTS.find( ( { from } ) =>
+			from.test( params.get( 'path' ) )
+		);
+
+		if ( ! matchedRedirect ) {
+			return;
+		}
+
+		params.set(
+			'path',
+			params
+				.get( 'path' )
+				.replace( matchedRedirect.from, matchedRedirect.to )
```

Comment:
Do we need a check for `.from` and `.to` just in case they weren't defined?

Reply:
I think it's fine at this scale. Maybe add some type checking if the list gets longer, but at this moment I only expect two or three entries.